examples/curb-detection/lifelong_learning_bench/testalgorithms/rfnet/RFNet/utils/metrics.py

```diff
@@ -134,7 +134,11 @@ def _generate_matrix(self, gt_image, pre_image):
         return confusion_matrix
 
     def add_batch(self, gt_image, pre_image):
+        gt_image = np.array(gt_image)
+        pre_image = np.array(pre_image)
         print(gt_image.shape, pre_image.shape)
+        if gt_image.shape != pre_image.shape:
+            pre_image = pre_image[0]
```

Comment:
Changing the dimension of "pre_image" if the condition "!=" is true is not always applicable. Users might have different ways to process pre_image or gt_image before passing them to this function. I suggest that you modify the dimension of "pre_image" outside this function.